server/db/models/ordersproducts.js

```diff
@@ -0,0 +1,24 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const OrdersProducts = db.define('ordersProducts', {
+  orderId: {
+    type: Sequelize.INTEGER,
+    allowNull: false
+  },
+  productId: {
+    type: Sequelize.INTEGER,
+    allowNull: false
+  },
+  quantity: {
+    type: Sequelize.INTENGER,
+    allowNull: false
+  },
+  purchasePrice: {
+    type: Sequelize.FLOAT
```

Comment:
Write a custom getter for purchase price that queries the purchase price from the Products table whenever it's blank, here

Reply:
(And write a test of this custom getter)